src/test/java/org/commcare/formplayer/tests/UserUtilsTests.java

```diff
@@ -16,4 +21,18 @@ public void testGetShortUsername() {
         assertEquals("dedicatedemployee@example.com",
                 UserUtils.getShortUsername("dedicatedemployee@example.com", "example"));
     }
+
+    @Test
+    public void testGetUserLocations() {
+        String test_loc_id = "test_location_id";
```

Comment:
variables need to be camelcased as per our coding standards for Java

Reply:
whoops you're right - going back and forth between python and java code had me confused